src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer
+    .filter(movie => {
+      const lowerTitle = movie.title.toLowerCase();
+      const lowerDesc = movie.description.toLowerCase();
+      const lowerQuery = query.toLowerCase().trim();
+
+      return (
+        lowerTitle.includes(lowerQuery.toLowerCase())
+        || lowerDesc.includes(lowerQuery.toLowerCase())
```

Comment:
These are already lowercase
```suggestion
        lowerTitle.includes(lowerQuery)
        || lowerDesc.includes(lowerQuery)
```

Reply:
I changed the vars several times to i forgot to cut those in the final one. Thank you